### PR TITLE
Release updates md2

### PR DESCRIPTION
**Pull request by mdeceglie**
- [x] Code changes are covered by tests
-  ~New functions added to `__init__.py`~
- [x] API.rst is up to date, along with other sphinx docs pages
- ~Example notebooks are rerun and differences in results scrutinized~
- [x] Updated changelog

### CODE REVIEW

File: rdtools/normalization.py [hunk 1]
```diff
@@ -14,20 +14,22 @@ class ConvergenceError(Exception):
...
-        Right-labeled time series of expected PV power.
+        Right-labeled time series of expected PV power. (Note: Expected energy
+        is not supported.)
     poa_global : pd.Series
         Right-labeled time series of plane-of-array irradiance associated with
         `expected_power`
-    pv_input : str
-        'power' or 'energy' to specify type of input used for pv parameter
+    pv_input : {'power' or 'energy'}
+        Specifies the type of input used for pv parameter. Default: 'power'
```
Comment (kanderso-nrel) [inline rdtools/normalization.py:32]:
```suggestion
    pv_input : {'power' or 'energy'}, default 'power'
        Specifies the type of input used for `pv` parameter.
```

File: rdtools/normalization.py [hunk 2]
```diff
@@ -14,20 +14,22 @@ class ConvergenceError(Exception):
...
-    Normalize pv output based on expected PV power.
+    Normalize PV power or energy based on expected PV power.
 
     Parameters
     ----------
     pv : pd.Series
         Right-labeled time series PV energy or power. If energy, should *not*
-        be cumulative, but only for preceding time step.
+        be cumulative, but only for preceding time step. Type (energy or power)
+        must be specified in the pv_input parameter.
```
Comment (kanderso-nrel) [inline rdtools/normalization.py:24]:
```suggestion
        must be specified in the `pv_input` parameter.
```

**Review by kanderso-nrel — COMMENTED** [review]
Took me a second to understand how to read "Estimated degradation rate in units percent/year relative to energy_normalized=1." -- would something like this be better?

> Estimated annual degradation rate as a fraction of year 0 system capacity [%/yr]

Changelog updates look good to me, thanks for merging those stragglers.

File: rdtools/normalization.py (hunk 1 above)
Comment (mdeceglie) [reply rdtools/normalization.py:32]: https://numpydoc.readthedocs.io/en/latest/format.html Section 4 seems to indicate that the default should be in the description. But maybe I'm misinterpreting?

Comment (kanderso-nrel) [reply rdtools/normalization.py:32]: Hmm I thought `paramname : type, default X` was fairly standard but I'm not finding many examples to support that in the big packages.  Only one I found so far is the `axis` parameter here, but it's not really apples to apples: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.shift.html

Okay with me to follow the numpydoc guidelines.  I do still think `pv` needs the backticks.

**Comment by mdeceglie** [comment]
Regarding the degradation rate,
> Estimated annual degradation rate as a fraction of year 0 system capacity [%/yr]

will work for ols and classical decomposition. But not for YOY with `recenter=False` If recenter is off, the result depends is strictly relative to normalized_energy=1. Is there a more clear way to describe that case?
```
In [1]: import pandas as pd
   ...: import rdtools
   ...: 
   ...: x = pd.Series([1, 0.9, 0.8, 0.7], index=pd.date_range('01/01/01', periods=4, freq="AS"))
   ...: 

In [2]: rdtools.degradation_year_on_year(x)
Out[2]: 
(-9.999999999999998, array([-10., -10.]), {'YoY_values': dt
  2002-01-01   -10.0
  2003-01-01   -10.0
  2004-01-01   -10.0
  Name: yoy, dtype: float64,
  'renormalizing_factor': 1.0,
  'exceedance_level': -10.000000000000009})

In [3]: rdtools.degradation_year_on_year(10*x)
Out[3]: 
(-9.999999999999998, array([-10., -10.]), {'YoY_values': dt
  2002-01-01   -10.0
  2003-01-01   -10.0
  2004-01-01   -10.0
  Name: yoy, dtype: float64,
  'renormalizing_factor': 10.0,
  'exceedance_level': -10.000000000000009})

In [4]: rdtools.degradation_year_on_year(10*x, recenter=False)
Out[4]: 
(-100.0, array([-100., -100.]), {'YoY_values': dt
  2002-01-01   -100.0
  2003-01-01   -100.0
  2004-01-01   -100.0
  Name: yoy, dtype: float64,
  'renormalizing_factor': 1.0,
  'exceedance_level': -100.0})
```

**Comment by mdeceglie** [comment]
Let's go with @kanderso-nrel 's suggestion and describe the `recenter=False` case in more detail in the description for that parameter.

**Review by kanderso-nrel — APPROVED** [review]
LGTM

**Pull request merged**